docs/actions/assert.md

```diff
@@ -0,0 +1,235 @@
+# Assertions
```

Comment:
Remove head title, there is a default one taken from the menu entry.

Reply:
this is a subtitle. The page contains two subtitles: assertions and validations